portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewFields.testcase

```diff
@@ -1499,4 +1813,95 @@ definition {
 		}
 	}
 
+	@description = "LPS-187522. Allow administrators to configure client extension filters on dataset views."
```

Comment:
You have to use the LPS from the parent story, in this case LPS-182558. Also, the description makes no sense with the name of the test.